recipes/cuda-cupti/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "cuda-cupti" %}
+{% set version = "12.0.90" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cupti/{{ platform }}/cuda_cupti-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 3c93a6a8c5eab3a56c1a0a514c0ac7d374807c70a9551155f089588b6c5e1d10  # [linux64]
+  sha256: bbabb508eb4fb0f9d98ac3ab4a40d0278de41cef95139c1bfe6436b716dd6cfc  # [ppc64le]
+  sha256: 33b3b06cf9838fa3289c7e40f143a8f58b941ccad93f9d0640a29bb83aa2152e  # [aarch64]
+  sha256: b65a34a3715e3ef2b835bdd36c43b21c20800325be1eda1e84ad4d80ee837c92  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+
+outputs:
+  - name: cuda-cupti
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcupti.so.*                # [linux]
+        - if not exist %LIBRARY_BIN%\cupti64_*.dll exit 1  # [win]
+    requirements:
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+    files:
+      - lib/*.so.*            # [linux]
+      - targets/*/lib/*.so.*  # [linux]
+      - Library\lib\*.dll     # [win]
+
+  - name: cuda-cupti-dev
+    test:
+      commands:
+        - test -f $PREFIX/lib/libnvperf_host.so        # [linux]
+        - test -f $PREFIX/lib/libnvperf_target.so        # [linux]
+        - if not exist %LIBRARY_LIB%\cupti.lib exit 1  # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - cuda-cupti >={{ version }}
+    files:
+      - lib/*.so                  # [linux]
+      - targets/*/lib/*.so        # [linux]
+      - include                   # [linux]
+      - targets/*/include         # [linux]
+      - samples
+      - doc
```

Comment:
Should this package ship 11 MB of docs and samples? Feels like that could be dropped with no loss. I don't think users look for docs / samples in conda environments, unless we think this is a crucial part of being a "full CTK."

Reply:
Offline choice was to ship the docs and samples in a `cuda-cupti-doc` package.